go.mod

```diff
@@ -4,6 +4,7 @@ go 1.19
 
 require (
 	github.com/hashicorp/terraform-plugin-framework v1.3.4
+	github.com/hashicorp/terraform-plugin-framework-timetypes v0.1.0
```

Comment:
You might be already thinking of this, but do we want to release a `v0.2.0` and include those upstream changes?

Reply:
Just released timetypes@v0.2.0 and will update this migration to remove (now extra) code.